source/broadband/webpa_notification.c

```diff
@@ -1283,12 +1390,31 @@ void processNotification(NotifyData *notifyData)
 			}
 	        		break;
 
+	        	case PARAM_NOTIFY_RETRY:
+	        	{
+	        		strcpy(dest, "event:SYNC_NOTIFICATION");
+				ret = processParamNotificationRetry(&cmc, &cid);
+	        		if (ret != WDMP_SUCCESS)
+	        		{
+	        			free(dest);
+	        			return;
+	        		}					
+	        		cJSON_AddNumberToObject(notifyPayload, "cmc", cmc);
+	        		cJSON_AddStringToObject(notifyPayload, "cid", cid);
+				WAL_FREE(cid);
+	        	}
+	        		break;					
+
 	        	default:
 	        		break;
 	        }
 
 	        stringifiedNotifyPayload = cJSON_PrintUnformatted(notifyPayload);
-	        WalPrint("stringifiedNotifyPayload %s\n", stringifiedNotifyPayload);
+		
+		if(notifyData->type == PARAM_NOTIFY_RETRY)
+			WalInfo("Sending retry sync notification, stringifiedNotifyPayload %s\n", stringifiedNotifyPayload);
```

Comment:
This log is confusing. better to change as "stringifiedNotifyPayload during sync notify retry is %s "